editor/styles/global.scss

```diff
@@ -310,10 +309,11 @@ to make sure it renders on top is a common fix */
   height: 29px;
   display: flex !important;
   justify-content: center;
+  box-shadow: 0 0 0 2px #0000001a;
```

Comment:
Copied from mapbox styles. It was a TIL for me that HEX codes can have an alpha channel.